### PR TITLE
First approach for process using scala standard library

### PR DESCRIPTION
**Pull request by BeniVF**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -71,7 +71,7 @@ lazy val site = project.in(file("site"))
 // General Settings
 lazy val commonSettings = Seq(
   scalaVersion := "2.13.1",
-  crossScalaVersions := Seq(scalaVersion.value, "2.12.10"),
```
Comment (JesusMtnez) [inline build.sbt]: Why disabling cross-compiling for `2.12.10`?

File: core/src/main/scala/c4s/process/Process.scala [hunk 2]
```diff
@@ -0,0 +1,105 @@
+package c4s.process
+
+import cats.implicits._
+import cats.effect._
+import fs2.{Stream, text}
+
+import io.chrisdavenport.log4cats.Logger
```
Comment (JesusMtnez) [inline core/src/main/scala/c4s/process/Process.scala]:
```suggestion
import cats.effect._
import cats.implicits._
import fs2.{Stream, text}
import io.chrisdavenport.log4cats.Logger
```

File: core/src/main/scala/c4s/process/ProcessResult.scala [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package c4s.process
+
+import cats.effect.Sync
+import cats.implicits._
+
+import fs2.{Stream, text}
```
Comment (JesusMtnez) [inline core/src/main/scala/c4s/process/ProcessResult.scala]:
```suggestion
import cats.effect.Sync
import cats.implicits._
import fs2.{Stream, text}
```

File: core/src/test/scala/c4s/process/ProcessSpec.scala [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+import cats.implicits._
+import cats.effect._
+
+import scala.concurrent.ExecutionContext
+
+import org.specs2.mutable.Specification
+
+import io.chrisdavenport.log4cats.testing.TestingLogger
+import io.chrisdavenport.log4cats.testing.TestingLogger._
```
Comment (JesusMtnez) [inline core/src/test/scala/c4s/process/ProcessSpec.scala]:
```suggestion
import cats.effect._
import cats.implicits._
import io.chrisdavenport.log4cats.testing.TestingLogger
import io.chrisdavenport.log4cats.testing.TestingLogger._ 
import org.specs2.mutable.Specification
import scala.concurrent.ExecutionContext
```

File: build.sbt (hunk 1 above)
Comment (BeniVF) [reply build.sbt]: maybe we could do it in a different `pr` because of `scala.collection.immutable.LazyList`

Comment (JesusMtnez) [reply build.sbt]: Perfect then!

**Pull request merged**